src/app/MainStore.ts

```diff
@@ -38,10 +49,12 @@ export class MainStore {
   @action
   loadPermissions() {
     const requestId = ++this.permissionsRequestCount;
-    this.cubaREST.getPermissions()
-      .then(action((perms: PermissionInfo[]) => {
+    this.cubaREST.getRoles()
```

Comment:
Will fail on current and previous versions of REST API. Should be changed once https://github.com/cuba-platform/cuba-rest-js/issues/20 is supported

Reply:
TODO added